html/js/apiCalls.js

```diff
@@ -147,47 +147,35 @@ function drawTable(contacts) {
 	return contacts;
 }
 
-// Delete contact we clicked on. 
-// Add contact with new information.
-// Search again.
-function editContact(e) {
-	getId(e);
-}
-
-function getId(element) {
-}
-
-function deleteContact() {
-
-}
-
-function openEditPopup(e) {
-	console.log(e);
```

Comment:
Deleted the lines above because they looked like they were not being used. Let me know if this was incorrect and should place them back.